### ISSUE TITLE
'Reply to' email cause error on Send and invalidates session - issue in 7.10.4 and 7.10.3 

### ISSUE THREAD

**Issue by LEAP-nishit** [salesagility/SuiteCRM#5692]
#### Issue

Sending an email by using the following works
Email -> Compose View -> Fill Form -> Send email. 

The following does not work,
Email -> View Email -> Click on Email -> Reply -> Fill Form -> Send email

The following error is generated:

![msg1](https://user-images.githubusercontent.com/9854372/38539608-3b3bde48-3cdc-11e8-8906-b22086a1378f.png)

In both cases the `From:` is set to the same value. 

#### Expected Behavior
Users should be able to replay to an email. The settings work when composing from scratch, fails on reply.  

#### Actual Behavior
A error message (shown above) is displayed, the following error is generated in `suitecrm.log`. 

```log
Tue Apr 10 16:21:04 2018 [18443][1][ERROR] Unable to find relationship emails_email_templates
Tue Apr 10 16:21:04 2018 [18443][1][ERROR] Unable to find relationship emails_email_templates
Tue Apr 10 16:21:04 2018 [18443][1][ERROR] Unable to find relationship emails_email_templates
Tue Apr 10 16:21:11 2018 [18444][1][ERROR] Unable to find relationship emails_email_templates
Tue Apr 10 16:21:11 2018 [18444][1][ERROR] Unable to find relationship emails_email_templates
Tue Apr 10 16:21:11 2018 [18444][1][ERROR] Unable to find relationship emails_email_templates
Tue Apr 10 16:21:11 2018 [18444][1][ERROR] Unable to find relationship emails_email_templates
Tue Apr 10 16:21:31 2018 [18331][1][ERROR] Unable to find relationship emails_email_templates
Tue Apr 10 16:21:31 2018 [18331][1][SECURITY] User Administrator attempted to send an email using incorrect email account settings in which they do not have access to.

```
#### Steps to Reproduce

1. Clean Install of SuiteCRM 7.10.4
2. Setup system email 
3. Setup outbound email
4. Create a case and send email to a client 
5. Wait for client to respond 
6. Open email and reply - Should fail and log you out. 
7. Log back in and Send a new email - should work. 
#### Context

Sending and receiving email for Cases using the CRM requires users to reply to customer responses. In 7.10.x this is not possible.  

#### Your Environment

* SuiteCRM Version used: 7.10.4
* Browser: Firefox 59.0.2 and Chrome 65.0.3325.181
* Environment name and version (e.g. MySQL, PHP 7): 
    * Outbound Email: AWS SES
    * Inbound Email: Office 365
    * MariaDB 10.0.34
    * php: PHP 7.0.28
* Operating System and version (e.g Ubuntu 16.04): Ubuntu 16.04.4 LTS

**Comment by Dillon-Brown** [comment]
Confirmed. Works when sending from "system" but not from personal outbound.

**Comment by PedroErnst** [comment]
Thanks @LEAP-nishit for reporting this.

I've been unable to reproduce this particular error.

In your steps you don't seem to mention setting up the Inbound account, did you do this and select the appropriate related outbound account like this?

![image](https://user-images.githubusercontent.com/34056696/38812127-4e57a88a-4183-11e8-80e1-41f04b15f5e3.png)

In any case I strongly agree that this issue needs to be solved, and I also think that this kind of issue shouldn't log the user out.

**Comment by tech-ch1** [comment]
I've been able to reproduce this bug.
It occurred when I tried to add a custom outgoing email account (which had disappeared after upgrade) to my user.
After that, replying to an email was disconnecting me.

I deleted the outgoing email account, re-set the common outgoing email account, recreated the outgoing email account.
I also went to the screen and fixed the "Outgoing SMTP Mail Server" which was wrong.

Now, the emails are sending… (but the From field is wrong, I'm posting a new issue)
